docs/app/partials/getting-started.tmpl.html

```diff
@@ -1,38 +1,84 @@
-         title="Link opens in a new window">read about the framework</a>.</em></p>
+    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to:</em>
+    <ul>
+      <li>watch the videos about <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
+               title="Link opens in a new window">Angular.js framework</a></li>
+      <li>
+        read the
+          <a href="https://www.google.com/design/spec/material-design/introduction.html" target="_blank"
+                 title="Link opens in a new window">Material Design </a> specifications for components, animations, styles, and layouts.
+      </li>
+    </ul>
 
-    <h2>How do I start?</h2>
+    <p></p>
+    <h2>How do I start with Angular Material?</h2>
     <ul style="margin-bottom: 2em;">
       <li><a href="http://codepen.io/collection/AxKKgY/" target="_blank"
-             title="Link opens in a new window">Fork a Codepen</a></li>
+             title="Link opens in a new window">Visit CodePen Community' </a></li>
+
+      <li><a href="https://github.com/angular/material-start/tree/es5-tutorial" target="_blank"
+                   title="Link opens in a new window">Learn using an 8-step Tutorial</a></li>
       <li><a href="https://github.com/angular/material-start" target="_blank"
-             title="Link opens in a new window">Clone a Github Starter Project</a></li>
-    </ul>
+             title="Link opens in a new window">Use the Github Starter Project</a></li>
 
-    <h3>Including Angular Material and its dependencies</h3>
-    <ul style="margin-bottom: 2em;">
-      <li><a href="https://github.com/angular/material#bower">Using Bower</a></li>
-      <li><a href="https://github.com/angular/material#cdn">Using a CDN</a> (example below)</li>
+      <li style="font-weight: 400;color: #106CC8;">Use 'Edit on CodePen' button on any of our Demos<br/>
+        <img src="https://cloud.githubusercontent.com/assets/210413/11568997/ed86795a-99b4-11e5-898e-1da172be80da.png" style="width:75%;margin: 10px 30px 0 0">
+      </li>
+
+      <li>Easily build a Material application from scratch (below)</li>
     </ul>
 
-    <iframe height='412' scrolling='no'
-            src='//codepen.io/team/AngularMaterial/embed/JYvaxJ/?height=412&theme-id=20485&default-tab=result'
-            frameborder='no' allowtransparency='true' allowfullscreen='true' style='width: 100%;'>
-      See the Pen <a href='http://codepen.io/team/AngularMaterial/pen/JYvaxJ/'>Angular Material Basic App</a>
-      by Angular Material (<a href='http://codepen.io/AngularMaterial'>@AngularMaterial</a>) on
-      <a href='http://codepen.io'>CodePen</a>.
+    <p></p>
+    <h3>Build a Material Application (blank shell)</h3>
+
+    Without installing <b>any</b> libraries locally, you can use Google's CDN to reference our online, cached Angular libraries.
+    Here is a CodePen Material applications that shows how to do that:
```

Comment:
I would rework this whole section as 
"See this example application on CodePen that loads the Angular Material library from the Google CDN."